sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html

```diff
@@ -209,11 +226,13 @@
     texInternalFormat: 'RGB565', texFormat: 'RGB', texType: 'UNSIGNED_BYTE',
     readFormat: 'RGBA', readType: 'UNSIGNED_BYTE',
     clearColor: [0.5, 0.7, 0.2, 0], readBackColor: [127, 178, 51, 0], tol: [17, 9, 17, 0],
+    implReadBackColor: [0.5, 0.7, 0.2, 0], implTol: [0.02, 0.02, 0.02, 0.02],
```

Comment:
Given this fact, should we remove this tests for IMPLEMENTATION_COLOR_READ_TYPE/FORMAT? Since the value is not consistent across drivers.